ibis/pandas/aggcontext.py

```diff
@@ -360,6 +367,14 @@ def agg(self, grouped_data, function, *args, **kwargs):
                 assert isinstance(function, str)
                 method = operator.methodcaller(function, *args, **kwargs)
 
+            max_lookback = self.max_lookback
+            if max_lookback is not None:
+                agg_method = method
+
+                def f(s):
```

Comment:
Let's give this a more meaningful name.

Reply:
How about `sliced_agg` or whatever you like?

Reply:
Changed to `sliced_agg` in next commit.